etcdctlv3/command/global.go

```diff
@@ -64,24 +65,25 @@ func mustClient(endpoints []string, dialTimeout time.Duration, cert, key, cacert
 	return client
 }
 
-func newClientCfg(endpoints []string, dialTimeout time.Duration, cert, key, cacert string) (*clientv3.Config, error) {
+func newClientCfg(endpoints []string, dialTimeout time.Duration, cert, key, cacert string, skipTLSVerify bool) (*clientv3.Config, error) {
 	// set tls if any one tls option set
 	var cfgtls *transport.TLSInfo
 	tls := transport.TLSInfo{}
 	var file string
-	if cert != "" {
-		tls.CertFile = cert
-		cfgtls = &tls
-	}
 
-	if key != "" {
-		tls.KeyFile = key
+	if skipTLSVerify {
+		tls.InsecureSkipVerify = true
 		cfgtls = &tls
-	}
+	} else {
+		if key != "" {
+			tls.KeyFile = key
+			cfgtls = &tls
+		}
 
-	if cacert != "" {
-		tls.CAFile = file
-		cfgtls = &tls
+		if cacert != "" {
+			tls.CAFile = file
+			cfgtls = &tls
+		}
```

Comment:
skip tls verify means do not verify the server cert when using secure connection.